qutebrowser/mainwindow/tabwidget.py

```diff
@@ -178,6 +202,9 @@ def get_tab_fields(self, idx):
         fields['perc_raw'] = tab.progress()
         fields['backend'] = objects.backend.name
         fields['private'] = ' [Private Mode] ' if tab.is_private else ''
+
+        fields['collapsed'] = ' [...] ' if tab.node.collapsed else ''
```

Comment:
```suggestion
        fields['collapsed'] = '[...] ' if tab.node.collapsed else ''
```
I think it's clearer without the preceding space.

Reply:
@arza-zara: But this would break consistency with the preceeding line (`' [Private Mode] '`).

Reply:
Audio field doesn't have it either:
https://github.com/qutebrowser/qutebrowser/blob/158b3be6ce48532b65457b17b7066850d16f6612/qutebrowser/mainwindow/tabwidget.py#L62-L63
Anyway, should be made customizable: #1578